env/numerical_compliance.txt

```diff
@@ -368,6 +383,11 @@ devices given as ULP values.
 | Correctly rounded
 | Correctly rounded
 
+| *OpExtInst* *length*
+| \<= 0.5 + 0.5 * ((0.5 * n) + (0.5 * (n - 1))) ulp, for gentype with vector width n
```

Comment:
Used `0.5` as first value for half, since fp16 extension defines sqrt() as being correctly rounded